src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int MAX_ARRAY_LENGTH = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
logic in `getStatistic()` method is way too complex for this task
try to simplify it
for example, split string that you read by line separator, you will have array of strings `{"supply,123", "buy,344", ...}`
later you can loop through this array and parse each string, calculating `totalSum` and `totalBuy` at the same time